ui/components/app/transaction-detail/transaction-detail.stories.js

```diff
@@ -21,7 +21,7 @@ const rows = [
     }
     detailText="0.00896 ETH"
     detailTotal="$15.73"
-    subTitle={<GasTiming text="Very likely in < 15 seconds" />}
+    subTitle={<GasTiming />}
```

Comment:
I guess we will need to update Storybook with the right props in some future PRs? Assuming that `maxPriorityFeePerGas` is mandatory.